components/filtergyms.vue

```diff
@@ -0,0 +1,326 @@
+            <input type="checkbox" class="custom-control-input" id="arts">
+            <label class="custom-control-label" for="arts">Material Arts</label>
+          </b-col>
+        </b-row>
+        <b-row class="checkboxes">
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="kickboxing">
+            <label class="custom-control-label" for="kickboxing">Kickboxing</label>
+          </b-col>
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="physiotherapy">
+            <label class="custom-control-label" for="physiotherapy">Physiotherapy</label>
+          </b-col>
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="basketball">
+            <label class="custom-control-label" for="basketball">Basketball</label>
+          </b-col>
+        </b-row>
+        <b-row class="checkboxes">
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="camp">
+            <label class="custom-control-label" for="camp">Boot Camp</label>
+          </b-col>
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="barre">
+            <label class="custom-control-label" for="barre">Barre</label>
+          </b-col>
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="soccer">
+            <label class="custom-control-label" for="soccer">Soccer</label>
+          </b-col>
+        </b-row>
+        <b-row class="checkboxes">
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="dancing">
+            <label class="custom-control-label" for="dancing">Dancing</label>
+          </b-col>
+          <b-col lg="4" sm="4" md="4" class="col-4">
+            <input type="checkbox" class="custom-control-input" id="climbing">
+            <label class="custom-control-label" for="climbing">Rock Climbing</label>
+          </b-col>
+          <b-col lg="4" sm="4" md="4" class="col-4"></b-col>
+        </b-row>
+        <center>
+          <button class="clear">CLEAR ALL</button>
+        </center>
+      </b-form>
+    </b-container>
+  </div>
+</template>
+
+<script>
+import { mdbInput } from "mdbvue";
+export default {
+  name: "Filtergyms",
+  components: {
+    mdbInput
+  }
+};
+</script>
+
+<style>
+body {
+  margin-bottom: 4% !important;
+  margin: 2% !important;
+}
+.header {
+  background-color: #162233;
+  color: white;
+}
+.imgs {
+  height: 15px;
+  margin-left: 7%;
+}
+.imd {
+  height: 15px;
+}
+.image {
+  float: right;
+  display: inline-flex;
+  padding-left: 15% !important;
+}
+.topheader {
+  padding-top: 0.5%;
+  width: 100%;
+  margin-left: 0px !important;
```

Comment:
why are we using important always. i see so many. important is to be used in rare case only